replica/replica.go

```diff
@@ -962,6 +969,23 @@ func (r *Replica) createDisk(name string, userCreated bool, created string) erro
 	r.volume.UserCreatedSnap = append(r.volume.UserCreatedSnap, userCreated)
 	r.activeDiskData = append(r.activeDiskData, &newHeadDisk)
 
+	// assign it to temp variable, so that	r.info remains
+	// intact in case of crash
```

Comment:
in case of crash, everything in-memory is gone :).. so, not to worry here to use r.info

Reply:
@vishnuitta i tested here by panic call and after panic it executes all the functions in stack and close is being called which modifies the content of volume.meta. That's why added this.